packages/editor/src/store/reducer.js

```diff
@@ -826,6 +827,26 @@ export function isReady( state = false, action ) {
 	return state;
 }
 
+/**
+ * Reducer returning the post editor setting.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+export function editorSettings( state = EDITOR_SETTINGS_DEFAULTS, action ) {
```

Comment:
The `editor` prefix always feels redundant. I know part of the idea here is to try to have some distinction between editor settings and block editor settings without calling both selector's `getSettings` (presumably because it's likely to conflict when using both in a single `mapSelectToProps` ?).

I wonder if:

- We just call the reducer `settings` even if the selector is still called `getEditorSettings` ?
- We're consistently explicit, i.e. `getEditorSettings` and `getBlockEditorSettings` ?
- We don't bother qualifying what type of setting it is, since it's clear by the module, and call both selectors `getSettings`? (Conflict potential be damned)

Reply:
Agreed, "editor" is redundant with the module itself. The only reason is backwards compatibility for me here.